### PR TITLE
Add e2e-prebuilt-kernel-module script

### PR DESCRIPTION
**Pull request by enriquebelarte**
Adds e2e test for prebuilt kernel module.

### CODE REVIEW

File: ci/module-kmm-ci.template.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  name: NAME_CHANGEME
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: KMOD_CHANGEME
+      imagePullPolicy: Never
+      kernelMappings:
+        - literal: KVER_CHANGEME
+          containerImage: kmm-kmod:local
```
Comment (qbarrand) [inline ci/module-kmm-ci.template.yaml:14]: We probably have to change this for an image inside the internal registry.

File: ci/prow/e2e-prebuilt-kernel-module [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+
+check_kmm_mod () {
+oc debug node/$NODE -T -- chroot /host sh -c "lsmod | grep kmm"
```
[the remark this answers is not in the document]
Comment (enriquebelarte) [reply ci/prow/e2e-prebuilt-kernel-module:7]: IMHO it is clearer this way what command is going to be run inside debug pod.

**Pull request merged**